### DISCUSSION TITLE
Get list of connected clients

### DISCUSSION THREAD

**Discussion by jk23909** [FreeOpcUa/python-opcua#1248] [Q&A]
How do I programmatically get a list of clients connected to my server?

**Comment by AndreasHeine** [comment]
unfortunately the session diagnostics part is not implementet in the python-opcua-server ... typically you can use the "SessionsDiagnosticsSummary" (Server -> ServerDiagnostics -> SessionsDiagnosticsSummary) to get the connected clients!
